src/java/org/apache/cassandra/db/streaming/CassandraOutgoingFile.java

```diff
@@ -127,6 +135,8 @@ public String getName()
     @Override
     public long getSize()
```

Comment:
nit: Perhaps we should just rename it `getEstimatedSize()`?

Reply:
+1